modules/mbedtls/Kconfig.tls-generic

```diff
@@ -559,6 +559,16 @@ config MBEDTLS_PSA_P256M_DRIVER_ENABLED
 	  Enable support for the optimized sofware implementation of the secp256r1
 	  curve through the standard PSA API.
 
+config MBEDTLS_PSA_KEY_SLOT_COUNT
+	int "Number of key slots in PSA core"
```

Comment:
```suggestion
	int "Number key slots in the PSA Crypto core"
```

Reply:
Why no "of" in the sentence? It seems correct to me: it's the number _of_ key slots...